test/capgen_test/temp_set.F90

```diff
@@ -56,6 +59,8 @@ SUBROUTINE temp_set_run(ncol, lev, timestep, temp_level, temp, ps,         &
        end do
     end do
 
+    var_array(:,:,:,:) = 1._kind_phys
+
```

Comment:
Should we add a test for the correct lower and upper bounds for soil_levs here (hardcoded, since we know what these bounds are)?

Reply:
Good idea. Done.